P0009/P0009.bs

```diff
@@ -2147,20 +2250,21 @@ template<class OtherElementType, class OtherExtents, class OtherLayoutPolicy, cl
   constexpr basic_mdspan& operator=(const basic_mdspan<OtherElementType, OtherExtents, OtherLayoutPolicy, OtherAccessor>& other);
 ```
 
+<!-- NOTE is_assignable_v<T,U> means T is assignable from U -->
+* *Constraints:*
+    + `is_assignable_v<ElementType(*)[], OtherElementType(*)[]>` is `true`, <!-- lifted directly from span wording -->
```

Comment:
Isn't the accessor pointer type assignment sufficient (below)? The above wording is only in `span` because it lacks the `Accessor` to constrain the pointer type, which could come from an allocator to a container that uses fancy pointers.  I don't think we have that problem.

Reply:
@dhollman I agree; this must be a left-over from the now deleted mdspan(span) constructor.